tests/plugins/test_flatpak_create_oci.py

```diff
@@ -595,7 +602,9 @@ def stream_to_generator(s):
             runner.run()
         assert expected_exception in str(ex.value)
     else:
+        assert workflow.builder.image_id == 'xxx'
         runner.run()
+        assert re.match(r'^sha256:\w{64}$', workflow.builder.image_id) is not None
```

Comment:
I got the impression from a review discussion a few weeks ago that the consensus was `is not None` should be avoided in cases that it isn't necessary?

Reply:
You are right. It is definitely not necessary here. At first, I thought this would improve test failures output, but it ends up I was wrong.